service/new_account.py

```diff
@@ -2,6 +2,7 @@
 from dto.license_dto import LicenseDTO
```

Comment:
the changes of this file is not nessasary

Reply:
pycharm need 2 blank line for every function, i see the yellow line i beh tahan